conduct.md

```diff
@@ -36,31 +36,39 @@ TODO: Write an introductory paragraph.
 
 ### Communication
 
-TODO: Write an introductory paragraph.
+Communication is the cornerstone of collaboration, and all communications for team-based activities in the Software Engineering course must adhere to the below guidelines for professionalism. To be clear, we recognize two platforms for our official communication: Discord and GitHub. That having been said, the below guidelines ought to be followed for any in-person communication pertaining to or within the classroom.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are encouraged to:
+
+- Be proactive about communicating. This can include utilizing whatever platforms necessary to let other know ahead of time of any kinds of changes, concerns, or ideas.
+
+- Constructively criticize others. For example, students should give honest and beneficial suggestions that do not alienate others and their work, which will encourage later participation.
+
+- Acknowledge others. Students should not isolate themselves or isolate others from any form of communication. Students should pull one another, and themselves, into the loop.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are discouraged to:
```

Comment:
Maybe 'discouraged from' would flow better? That is personal preference though, if you want to keep it as is thats fine!